content/app/app-dev-course/_index.nb.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Introkurs
+description: Et kursopplegg for å lære seg apputvikling med Altinn Studio og lokale verktøy
+linktitle: Introkurs
+tags: [apps, build, deploy, training]
+weight: 15
+---
+
+Dette intro-kurset er delt opp i seks bolker og dekker et fiktivt case for Sogndal kommune.
+I løpet av disse seks bolkene vil du ha vært innom den mest sentrale funksjonaliteten for en Altinn App,
+samt testing av applikasjonen lokalt og i testmiljø.
+
+
+Tre ting må være på plass før du går i gang med dette kurset.
+
+1. [En bruker i Altinn Studio](../getting-started/first-time-setup/#lag-en-bruker-i-altinn-studio)
+2. [Ditt lokale utviklingsmiljø er satt opp med støtte for utvikling og testing av apps](https://github.com/Altinn/altinn-studio/blob/master/LOCALAPP.md#prerequisites)
+3. [Tilgang til en organisasjon med et testmiljø](../getting-started/first-time-setup/#bli-del-av-en-organisasjon)*
```

Comment:
typo on the * and \* in the next line ?

Reply:
meant for extra information about the list item with the *